app/app.tsx

```diff
@@ -70,10 +70,12 @@ function App() {
 
   useEffect(() => {
     // Activate live activities listener on iOS 16.2+
-    if (canRunLiveActivities) {
-      monitorLiveActivities()
-    }
-  }, [])
+    rootStore?.ride.checkLiveActivitiesSupported().then((canRunLiveActivities) => {
+      if (canRunLiveActivities === true) {
```

Comment:
```suggestion
      if (canRunLiveActivities) {
```

Reply:
I like it better when it's explicit, more readable and the intention is clearer

Reply:
It's not a conversion we currently have in the app. We don't have this anywhere in our source - but it's your call. If you like it feel free to resolve this.